src/java/org/apache/cassandra/db/rows/Cell.java

```diff
@@ -282,6 +301,9 @@ public <V> Cell<V> deserialize(DataInputPlus in, LivenessInfo rowLiveness, Colum
                 }
             }
 
+            if (localDeletionTime < 0)
+                localDeletionTime = helper.version < MessagingService.VERSION_42 ? LEGACY_OVERFLOWED_DELETION_TIME : Cell.deletionTimeGuavaUintToLong((int) localDeletionTime);
```

Comment:
Nit: break the long line
```suggestion
                localDeletionTime = helper.version < MessagingService.VERSION_42
                                    ? LEGACY_OVERFLOWED_DELETION_TIME
                                    : Cell.deletionTimeGuavaUintToLong((int) localDeletionTime);
```